public/app/assetlibrary/assetLibraryFactory.js

```diff
@@ -65,10 +65,6 @@
      * @param  {Number}               [searchOptions.user]            The id of the user who created the assets
      * @param  {String}               [searchOptions.section]         The name of section (i.e., subset of users) to filter assets by
      * @param  {String}               [searchOptions.type]            The type of assets
-     * @param  {Boolean}              [filters.hasComments]           If true then exclude zero comment_count; if false then zero comment_count only; if null do nothing
-     * @param  {Boolean}              [filters.hasImpact]             If true then exclude zero impact; if false then zero impact only; if null do nothing
-     * @param  {Boolean}              [filters.hasLikes]              If true then exclude zero likes; if false then zero likes only; if null do nothing
-     * @param  {Boolean}              [filters.hasViews]              If true then exclude zero views; if false then zero views only; if null do nothing
```

Comment:
`assetLibraryFactory` does not receive filters above, it generates them with `utilService.addAssetInclusionFilter(searchOptions)` (follow-up to PR #400)